### PR TITLE
Just log messages so you can see why it fails

### PR DESCRIPTION
**Pull request by ScyDev**

### CODE REVIEW

**Comment by zenweasel** [comment]
We don't want to pass this info to the client, so we can't accept the `console.log` statements. The `ReactionCore.Log` statements are fine.

File: server/braintree.js [hunk 1]
```diff
@@ -25,6 +28,8 @@ Meteor.methods({
    * @return {Object} results - Object containing the results of the transaction
    */
   "braintreeSubmit": function (transactionType, cardData, paymentData) {
+    ReactionCore.Log.info("braintreeSubmit: ",transactionType, " ", cardData, " ", paymentData);
+
```
Comment (zenweasel) [inline server/braintree.js]: We absolutely **DO NOT** want to log `cardData` as this exposes sensitive data.

Comment (ScyDev) [inline server/braintree.js]: You are absolutely correct of course! Didn't think of that at the time.

**Comment by zenweasel** [comment]
These statements are good for debugging but wouldn't be appropriate for a production environment. Closing this PR but will open an issue to add more logging to this (and other) payment packages.

**Pull request closed without merging**